index.html

```diff
@@ -293,3 +293,27 @@ <h2>FAQ</h2>
wLDEuNi0wLjMsMi0xLjFMMjE4LjIsMjIuN0wyMTguMiwyMi43eg0KCQkgTTIxNS4zLDIxYzAtMSwxLjgtMC44LDIuOC0xLjJ2MC40YzAsMS0wLjksMS42LTEuOCwxLjZDMjE1LjgsMjEuOCwyMTUuMywyMS41LDIxNS4zLDIxeiBNMjI3LjEsMTkuN2MwLTEuOC0xLjItMy4xLTIuOS0zLjENCgkJYy0wLjgsMC0xLjUsMC40LTIsMXYtMy45SDIyMXY5aDEuMnYtMC45YzAuNSwwLjYsMS4yLDEsMiwxQzIyNS44LDIyLjgsMjI3LjEsMjEuNSwyMjcuMSwxOS43eiBNMjI0LDE3LjdjMSwwLDEuOCwwLjgsMS44LDINCgkJYzAsMS4yLTAuOCwyLTEuOCwyYy0xLDAtMS44LTAuOC0xLjgtMkMyMjIuMiwxOC41LDIyMywxNy43LDIyNCwxNy43eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xNi4xLDBMMTYuMSwwaDYuNnY1LjdoLTYuNmMtMSwwLTEuOSwwLjgtMS45LDEuOXYxNS4xSDguNVY3LjZsMCwwYzAtMSwwLjgtMS45LDEuOS0xLjloMC41aDEuNA0KCQljMSwwLDEuOS0wLjgsMS45LTEuOVYxLjlDMTQuMiwwLjgsMTUsMCwxNi4xLDB6Ii8+DQoJPHBvbHlnb24gY2xhc3M9InN0MCIgcG9pbnRzPSI4LjUsNS43IDAsNS43IDAsMCA4LjUsMCAJIi8+DQoJPHBhdGggY2xhc3M9InN0MCIgZD0iTTUzLjksMTUuMVYwaC01Ljd2MTdINTJDNTMsMTcsNTMuOSwxNi4yLDUzLjksMTUuMUw1My45LDE1LjF6Ii8+DQoJPHBhdGggY2xhc3M9InN0MCIgZD0iTTMzLjEsMjIuN0wzMy4xLDIyLjdoMy44VjE3aC0zLjhjLTEsMC0xLjktMC44LTEuOS0xLjlWMGgtNS43djE1LjFsMCwwYzAsMSwwLjgsMS45LDEuOSwxLjloMC41aDEuNA0KCQljMSwwLDEuOSwwLjgsMS45LDEuOXYxLjlDMzEuMiwyMS44LDMyLDIyLjcsMzMuMSwyMi43eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik00NC40LDIyLjdMNDQuNCwyMi43aDMuOFYxN2gtMy44Yy0xLDAtMS45LTAuOC0xLjktMS45VjBoLTUuN3YxNS4xbDAsMGMwLDEsMC44LDEuOSwxLjksMS45aDAuNWgxLjQNCgkJYzEsMCwxLjksMC44LDEuOSwxLjl2MS45QzQyLjUsMjEuOCw0My40LDIyLjcsNDQuNCwyMi43eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xMDQuOSw3LjZ2MTUuMWgtNS43di0xN2gzLjhDMTA0LDUuNywxMDQuOSw2LjUsMTA0LjksNy42eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik04OS44LDBMODkuOCwwaDcuNmMxLDAsMS45LDAuOCwxLjksMS45djMuOGgtOS40Yy0xLDAtMS45LDAuOC0xLjksMS45djE1LjFoLTUuN1Y3LjZsMCwwYzAtMSwwLjgtMS45LDEuOS0xLjkNCgkJaDAuNUg4NmMxLDAsMS45LTAuOCwxLjktMS45VjEuOUM4Ny45LDAuOCw4OC43LDAsODkuOCwweiIvPg0KCTxyZWN0IHg9Ijg2LjIiIHk9IjEzLjIiIGNsYXNzPSJzdDAiIHdpZHRoPSIxNS44IiBoZWlnaHQ9IjMuOCIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik02NC4zLDIyLjdMNjQuMywyMi43aDE1LjFWMTdINjQuM2MtMSwwLTEuOS0wLjgtMS45LTEuOXYtMS45VjUuN2gtMy44Yy0xLDAtMS45LDAuOC0xLjksMS45djUuN3YxLjlsMCwwDQoJCWMwLDEsMC44LDEuOSwxLjksMS45aDAuNWgxLjRjMSwwLDEuOSwwLjgsMS45LDEuOXYxLjlDNjIuNCwyMS44LDYzLjIsMjIuNyw2NC4zLDIyLjdMNjQuMywyMi43eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik03OS40LDBINjQuM2MtMSwwLTEuOSwwLjgtMS45LDEuOXYzLjhoMTdDNzkuNCw1LjcsNzkuNCwwLDc5LjQsMHoiLz4NCgk8cG9seWdvbiBjbGFzcz0ic3QwIiBwb2ludHM9Ijc2LjUsMTMuMiA2MC40LDEzLjIgNjAuNCw5LjQgNzYuNSw5LjQgCSIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xMTUuMywyMi43TDExNS4zLDIyLjdoMTAuNGMxLDAsMS45LTAuOCwxLjktMS45VjE3aC0xMi4zYy0xLDAtMS45LTAuOC0xLjktMS45di0xLjlWNS43aC0zLjgNCgkJYy0xLDAtMS45LDAuOC0xLjksMS45djUuN3YxLjlsMCwwYzAsMSwwLjgsMS45LDEuOSwxLjloMC41aDEuNGMxLDAsMS45LDAuOCwxLjksMS45djEuOUMxMTMuNCwyMS44LDExNC4yLDIyLjcsMTE1LjMsMjIuNw0KCQlMMTE1LjMsMjIuN3oiLz4NCgk8cGF0aCBjbGFzcz0ic3QwIiBkPSJNMTMwLjQsMGgtMTUuMWMtMSwwLTEuOSwwLjgtMS45LDEuOXYzLjhoMTdMMTMwLjQsMEwxMzAuNCwweiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xMjcuNiwxMy4yaC04LjVWOS40aDEyLjNjMSwwLDEuOSwwLjksMS45LDEuOXYxLjlWMTdoLTUuN0MxMjcuNiwxNywxMjcuNiwxMy4yLDEyNy42LDEzLjJ6Ii8+DQo8L2c+DQo8L3N2Zz4NCg=="/>
```

Comment:
Is there a reason these are inlined, as opposed to being stored as an image file?

Reply:
No specific reason, I'll update